quiche/src/lib.rs

```diff
@@ -2591,6 +2591,21 @@ impl Connection {
 
                         stream.send.ack_and_drop(offset, length);
 
+                        qlog_with_type!(QLOG_DATA_MV, self.qlog, q, {
+                            let ev_data = EventData::DataMoved(
+                                qlog::events::quic::DataMoved {
+                                    stream_id: Some(stream_id),
+                                    offset: Some(offset),
+                                    length: Some(length as u64),
+                                    from: Some(DataRecipient::Transport),
+                                    to: Some(DataRecipient::Dropped),
+                                    data: None,
+                                },
+                            );
+                            let now = time::Instant::now();
```

Comment:
It looks like we already have a `now` variable used for qlog above?

Reply:
good catch, fixed